src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -67,6 +68,25 @@ const EventEntries = createReactClass({
     };
   },
 
+  recordSourcemapError(errorType) {
+    analytics('sourcemap.sourcemap_error', {
```

Comment:
nit: one cool thing you could do here is take advantage of es6 destructuring and do 
` let {project, event} = this.props;` which will let you pull just `project.organization.id` and `event.groupID`